### ISSUE TITLE
layer_concatonate example

### ISSUE THREAD

**Issue by cj-wilson** [rstudio/keras#113]
What is the proper usage for layer_concatonate()?  I have two models I'd like to merge and I'm getting the error below.  Model params farther down.  Thank you in advance.

-Chris

model = keras_model_sequential()
model %>%  
  layer_concatenate(inputs = list(dense_branch, lstm_branch), axis = -1L) 

Error in layer_concatenate(., inputs = list(dense_branch, lstm_branch),  : 
  unused argument (.)

> summary(lstm_branch)
Model
________________________________________________________________________________________________________________________
Layer (type)                                          Output Shape                                    Param #           
========================================================================================================================
embedding_1 (Embedding)                               (None, None, 64)                                64000             
________________________________________________________________________________________________________________________
lstm_1 (LSTM)                                         (None, 64)                                      33024             
========================================================================================================================
Total params: 97,024
Trainable params: 97,024
Non-trainable params: 0
________________________________________________________________________________________________________________________

> summary(dense_branch)
Model
________________________________________________________________________________________________________________________
Layer (type)                                          Output Shape                                    Param #           
========================================================================================================================
dense_1 (Dense)                                       (None, 128)                                     5376              
________________________________________________________________________________________________________________________
dropout_1 (Dropout)                                   (None, 128)                                     0                 
________________________________________________________________________________________________________________________
dense_2 (Dense)                                       (None, 64)                                      8256              
========================================================================================================================
Total params: 13,632
Trainable params: 13,632
Non-trainable params: 0
____________________________

**Comment by jjallaire** [comment]
Here are some links to articles/examples that illustrate various uses of `layer_concatenate`:

https://rstudio.github.io/keras/articles/about_keras_models.html
https://rstudio.github.io/keras/articles/functional_api.html
https://rstudio.github.io/keras/articles/examples/babi_memnn.html
https://rstudio.github.io/keras/articles/examples/variational_autoencoder.html

**Comment by tybyers** [comment]
Updated links for the above:
https://keras.rstudio.com/articles/about_keras_models.html
https://keras.rstudio.com/articles/functional_api.html
https://keras.rstudio.com/articles/examples/babi_memnn.html
https://keras.rstudio.com/articles/examples/variational_autoencoder.html
